### PR TITLE
SuperPMI: Support GetHFAType for ARM32 hardFP

### PR DESCRIPTION
**Pull request by hseok-oh**
Related issue: #7871 

- Implementation of GetHFAType in SuperPMI

- Add flag for ARM32

@bruceforstall PTAL

### CODE REVIEW

File: src/ToolBox/superpmi/superpmi-shared/methodcontext.cpp [hunk 1]
```diff
@@ -2842,6 +2842,34 @@ CORINFO_CLASS_HANDLE MethodContext::repGetArgClass(CORINFO_SIG_INFO* sig, CORINF
...
+        GetHFAType = new LightWeightMap<DWORDLONG, DWORD>();
+
+    GetHFAType->Add((DWORDLONG)clsHnd, (DWORD)result);
+    DEBUG_REC(dmpGetHFAType((DWORDLONG)clsHnd, (DWORD)result));
+    return;
+}
+
+void MethodContext::dmpGetHFAType(DWORDLONG key, DWORD value)
+{
+    printf("GetHFATypet key %016llX, value %u ", key, value);
```
Comment (BruceForstall) [inline src/ToolBox/superpmi/superpmi-shared/methodcontext.cpp]: minor: typo: GetHFATypet => GetHFAType

**Comment by BruceForstall** [comment]
LGTM. Thanks!

**Comment by hseok-oh** [comment]
@dotnet-bot test Linux ARM Emulator Cross Release Build please

**Comment by hseok-oh** [comment]
@dotnet-bot test Ubuntu x64 Checked Build and Test please

**Comment by BruceForstall** [comment]
Ubuntu x64 failed due to:
```
FAILED   - GC/LargeMemory/API/gc/keepalive/keepalive.sh
               BEGIN EXECUTION
               /mnt/j/workspace/dotnet_coreclr/master/checked_ubuntu_tst_prtest/bin/tests/Windows_NT.x64.Checked/Tests/coreoverlay/corerun keepalive.exe 2048
               
               Unhandled Exception: System.IO.IOException: The process cannot access the file '/proc/meminfo' because it is being used by another process.
                  at System.IO.FileStream.Init(FileMode mode, FileShare share)
                  at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
                  at System.IO.File.Open(String path, FileMode mode, FileAccess access)
                  at System.IO.StreamReader..ctor(String path, Encoding encoding, Boolean detectEncodingFromByteOrderMarks, Int32 bufferSize)
                  at System.IO.File.InternalReadAllLines(String path, Encoding encoding)
                  at MemCheck.GetPhysicalMem()
                  at KeepAliveTest.Main(String[] args)
               ./keepalive.sh: line 124: 61148 Aborted                 (core dumped) $_DebuggerFullPath "$CORE_ROOT/corerun" keepalive.exe $CLRTestExecutionArguments
               Expected: 100
               Actual: 134
               END EXECUTION - FAILED
```

**Comment by BruceForstall** [comment]
Try again:

@dotnet-bot test Ubuntu x64 Checked Build and Test

**Pull request merged**